addons/deleted-comments/style.css

```diff
@@ -0,0 +1,3 @@
+.removed {
+  display: block !important;
```

Comment:
Oh, and please use a more specific selector. This will match anything with class removed, not only removed comments. Let's be more specific just in case :P

Reply:
ok